frontend/src/scenes/session-recordings/player/controller/PlayerSeekbarPreview.tsx

```diff
@@ -125,3 +125,5 @@ export function PlayerSeekbarPreview({ minMs, maxMs, seekBarRef, activeMs }: Pla
         </div>
     )
 }
+
+export const PlayerSeekbarPreview = memo(_PlayerSeekbarPreview)
```

Comment:
this was re-rendering constantly because the seekbar renders on every tick but it definitely doesn't need to